settings.gradle

```diff
@@ -5,5 +5,5 @@ include 'buildSrc'
 include 'frontend-service-client'
 include 'frontend-service'
 include 'frontend-infrastructure'
-include 'data-service-model'
+include 'data-service-client'
```

Comment:
Yea we have two options: have two packages - one for the model and one for the client or put them both in the same package. Here I was going for the same package. The advantage of separating them would be the dataservice won't have the client file. Either way is fine by me at this point. You have a preference?